src/lookup-table/controllers/ImportCsvUpdate.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { parseStream } from 'fast-csv'
+import { LookupTableService } from '../services'
+
+const ImportCsvUpdateController = async (request: any, reply: any) => {
+  const lookupTableId = Number(request.params.lookupTableId)
+  const data = await request.file()
+
+  const lookupTableService = LookupTableService({ tableId: lookupTableId })
+
+  let stream = await parseStream(data.file, {
+    headers: lookupTableService.parseCsvHeaders,
+  })
+    .on('data', async (row: any) => {
+      stream.pause()
```

Comment:
@andreievg actually when I did not use pause earlier, I found that since the calls are housed inside event listeners. In some cases, the rows were getting pushed onto the array out of order. I used `pause` and `resume` calls to be safe.

Reply:
Was this before refactor ? In the previous version there was a DB call (which would cause order mismatch)
I dont' think order should be important, user may decide to sort csv before re-importing as update